hoot-core/src/main/cpp/hoot/core/conflate/highway/HighwayMatch.cpp

```diff
@@ -165,7 +165,7 @@ map<QString, double> HighwayMatch::getFeatures(const ConstOsmMapPtr& m) const
 set<pair<ElementId, ElementId>> HighwayMatch::getMatchPairs() const
 {
   set<pair<ElementId, ElementId>> result;
-  result.emplace(pair<ElementId, ElementId>(_eid1, _eid2));
+  result.emplace(_eid1, _eid2);
```

Comment:
@bwitham This I didn't know.  `std::set<>::emplace()` takes on the parameters of the constructor so that the object is constructed in place and doesn't require a temporary object before constructing another in the `set`.

Reply:
That's interesting.